compiler/optimizer/OMRTransformUtil.cpp

```diff
@@ -406,10 +406,15 @@ OMR::TransformUtil::createConditionalAlternatePath(TR::Compilation* comp,
    {
    cfg->setStructure(0);
 
+   bool mergeToElseBlock = elseBlock == mergeBlock ? true : false;
```

Comment:
Why not just:

```suggestion
   bool mergeToElseBlock = elseBlock == mergeBlock;
```
Or maybe:
```suggestion
   bool mergeToElseBlock = (elseBlock == mergeBlock);
```